apps/cli-daemon/src/app/utils/extract -value-object.ts

```diff
@@ -0,0 +1,49 @@
+import { JsonObject, JsonValue } from '@angular-devkit/core/src/json/utils';
```

Comment:
something wrong in the file name of this file and the spec

Reply:
remove the whitespace